packages/stream-tests/src/__tests__/model-instance-document.test.ts

```diff
@@ -514,16 +504,32 @@ describeIfV3ShouldPass('ModelInstanceDocument API multi-node tests', () => {
   let midMetadata: ModelInstanceDocumentMetadataArgs
 
   beforeAll(async () => {
-    ipfs0 = await createIPFS()
-    ipfs1 = await createIPFS()
+    ipfs0 = await createIPFS({
+      rust: {
+        type: 'binary',
```

Comment:
Can we move this logic into `createIPFS` as the default behavior so we don't need to update the call sites like this?

Reply:
Ready